docs/guide/slots.md

```diff
@@ -281,3 +281,16 @@ Filling in the `visual` slot is done by calling the appropriate slot method:
   <% end >
 <% end %>
 ```
+
+To test whether a polymorphic slot has been passed to the component, use the provided `#{slot_name}?` method.
+
+```erb
+<% if visual? %>
+  <div>
+    <div><%= content %></div>
+    <%= visual %>
+  </div>
+<% else %>
+  <div>content</div>
+<% end %>
```

Comment:
```suggestion
<% if visual? %>
  <%= visual %>
<% else %>
  <span class="visual-placeholder">N/A</span>
<% end %>
```

Minor, but thoughts on simplifying this example to remove `content` from the equation (we also use `content` in two different ways in the current diff)?